src/textdescriptives/components/quality_data_classes.py

```diff
@@ -24,11 +24,13 @@ class Config:
         extra = Extra.forbid
 
     threshold: Union[Interval, bool, None]
-    value: float
+    value: Union[float, None]
 
     @property
     def passed(self) -> bool:
         """Return True if the value is within the thresholds."""
+        if self.value is None:
+            return True
```

Comment:
```suggestion
            return None
```
I think this is better. It makes it clear that no check was made. We can then change the global class to allow  for None.